pipelines/shopify_dlt/__init__.py

```diff
@@ -0,0 +1,72 @@
+
+"""Fetches Shopify Orders and Products."""
+
+from typing import Any, Dict, Iterator, Iterator
+
+import shopify
+
+import dlt
+
+from dlt.extract.source import DltResource
+
+
+@dlt.source(name="shopify")
+def shopify_source(
+    private_app_password: str = dlt.secrets.value,
+    api_version: str = dlt.config.value,
+    shop_url: str = dlt.config.value,
+) -> Iterator[DltResource]:
+    """
+    The source for the shopify pipeline. Available resources are products, orders and customers.
+    @:param: private_app_password - The app password to the app on your shop
+    @:param: api_version - The api version to use
+    @:param: shop_url - The url of your shop
+    @:return: data_resources_list - list of dlt resources
+    """
+
+    # build client
+    session = shopify.Session(shop_url, api_version, private_app_password)
+    shopify.ShopifyResource.activate_session(session)
+
+    # define resources
+    @dlt.resource(primary_key="id", write_disposition="merge")
+    def products(updated_at = dlt.sources.incremental("updated_at", initial_value="1970-05-04")) -> Iterator[Dict[str, Any]]:
```

Comment:
`updated_at` is a ISO datetime string, right? then  good.

Reply:
Yes